### PR TITLE
move globals_whitelist to own module

### PR DESCRIPTION
**Pull request by mbargull**
Ah follow-up to gh-67:
It's a little easier to see which built-ins we allow and which not by listing them explicitly.
I moved this to a separate module to not clutter the main one.

### CODE REVIEW

**Comment by mbargull** [comment]
I commented the entries that we either shouldn't or likely not want with `# # ...`.
The entries with single `#` could be added safely, I guess.
Please edit this PR/the list as you see fit!

**Comment by mbargull** [comment]
> Please edit this PR/the list as you see fit!

And keep the `README.md` in sync ;).

File: vembrane/globals_whitelist.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+# The builtins list below was generated with:
+#    python -c 'print(
+#        *sorted(o for o in dir(__builtins__) if o.islower() and not o.startswith("_")),
+#        sep="\n",
+#    )'
+_builtins = {
+    obj.__name__: obj
+    for obj in (
+        # abs,
```
Comment (tedil) [inline vembrane/globals_whitelist.py:12]:
```suggestion
        abs,
```

File: vembrane/globals_whitelist.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        all,
+        any,
+        # # ascii,
+        # bin,
+        # bool,
```
Comment (tedil) [inline vembrane/globals_whitelist.py:17]:
```suggestion
        bool,
```

File: vembrane/globals_whitelist.py [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+        # # breakpoint,
+        # # bytearray,
+        # # bytes,
+        # # callable,
+        # chr,
```
Comment (tedil) [inline vembrane/globals_whitelist.py:22]:
```suggestion
        chr,
```

File: vembrane/globals_whitelist.py [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+        # # classmethod,
+        # # compile,
+        # # complex,
+        # # copyright,
+        # # credits,
+        # # delattr,
+        dict,
+        # # dir,
+        # divmod,
+        # enumerate,
```
Comment (tedil) [inline vembrane/globals_whitelist.py:32]:
```suggestion
        enumerate,
```

File: vembrane/globals_whitelist.py [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+        # # eval,
+        # # exec,
+        # # exit,
+        # filter,
```
Comment (tedil) [inline vembrane/globals_whitelist.py:36]:
```suggestion
        filter,
```

File: vembrane/globals_whitelist.py [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+        # # hash,
+        # # help,
+        # hex,
+        # # id,
+        # # input,
+        int,
+        # isinstance,
+        # # issubclass,
+        # iter,
+        # len,
```
Comment (tedil) [inline vembrane/globals_whitelist.py:52]:
```suggestion
        len,
```

File: vembrane/globals_whitelist.py [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+        # # hasattr,
+        # # hash,
+        # # help,
+        # hex,
+        # # id,
+        # # input,
+        int,
+        # isinstance,
+        # # issubclass,
+        # iter,
```
Comment (tedil) [inline vembrane/globals_whitelist.py:51]:
```suggestion
        iter,
```

File: vembrane/globals_whitelist.py [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+        # len,
+        # # license,
+        list,
+        # # locals,
+        map,
+        max,
+        # # memoryview,
+        min,
+        # next,
```
Comment (tedil) [inline vembrane/globals_whitelist.py:60]:
```suggestion
        next,
```

File: vembrane/globals_whitelist.py [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+        # # object,
+        # oct,
+        # # open,
+        # ord,
```
Comment (tedil) [inline vembrane/globals_whitelist.py:64]:
```suggestion
        ord,
```

File: vembrane/globals_whitelist.py [hunk 10]
```diff
@@ -0,0 +1,108 @@
...
+        # pow,
+        # # print,
+        # # property,
+        # # quit,
+        # range,
```
Comment (tedil) [inline vembrane/globals_whitelist.py:69]:
```suggestion
        range,
```

File: vembrane/globals_whitelist.py [hunk 11]
```diff
@@ -0,0 +1,108 @@
...
+        # # repr,
+        # reversed,
```
Comment (tedil) [inline vembrane/globals_whitelist.py:71]:
```suggestion
        reversed,
```

File: vembrane/globals_whitelist.py [hunk 12]
```diff
@@ -0,0 +1,108 @@
...
+        # round,
```
Comment (tedil) [inline vembrane/globals_whitelist.py:72]:
```suggestion
        round,
```
(if there's no round in math, didn't check)

File: vembrane/globals_whitelist.py [hunk 13]
```diff
@@ -0,0 +1,108 @@
...
+        set,
+        # # setattr,
+        # # slice,
+        # sorted,
```
Comment (tedil) [inline vembrane/globals_whitelist.py:76]:
```suggestion
        sorted,
```

File: vembrane/globals_whitelist.py (hunk 12 above)
Comment (tedil) [reply vembrane/globals_whitelist.py:72]: Just checked, there isn't.

**Comment by mbargull** [comment]
@tedil ;P
> > Please edit this PR/the list as you see fit!
> 
> And keep the `README.md` in sync ;).

**Pull request merged**